cns/kubecontroller/nodenetworkconfig/conversion_windows_test.go

```diff
@@ -13,15 +13,11 @@ var validOverlayRequest = &cns.CreateNetworkContainerRequest{
 			PrefixLength: uint8(subnetPrefixLen),
 			IPAddress:    primaryIP,
 		},
-		GatewayIPAddress: "10.0.0.0",
+		GatewayIPAddress: "10.0.0.1",
 	},
 	NetworkContainerid:   ncID,
 	NetworkContainerType: cns.Docker,
 	SecondaryIPConfigs: map[string]cns.SecondaryIPConfig{
-		"10.0.0.1": {
-			IPAddress: "10.0.0.1",
-			NCVersion: version,
-		},
 		"10.0.0.2": {
 			IPAddress: "10.0.0.2",
 			NCVersion: version,
```

Comment:
commenting on this line because github doesn't let me comment on an arbitrary line, but for LL25-29, I think if you make the change I suggest above where you skip the last IP in the block, you'd remove LL25-29

Reply:
removed those lines since we're skipping last IP of subnet now